nbconvert/filters/markdown_mistune.py

```diff
@@ -29,6 +35,31 @@ class MathInlineGrammar(mistune.InlineGrammar):
     text = re.compile(r'^[\s\S]+?(?=[\\<!\[_*`~$]|https?://| {2,}\n|$)')
 
 
+class MathBlockLexer(mistune.BlockLexer):
```

Comment:
This should have a comment describing the role of the `MathBlockLexer` as a means of preëmpting mistune's block grammar rules. It should also describe its general role as a pass-through, delegating the parsing to the `MathBlockLexer`.